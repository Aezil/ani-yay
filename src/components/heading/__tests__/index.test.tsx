import { render } from '@testing-library/react';
import React from 'react';

import { Heading } from '..';

describe('Heading', () => {
  it('should render Heading component', () => {
    const { container } = render(<Heading>I am a test</Heading>);
    expect(container).toMatchSnapshot();
  });
});
