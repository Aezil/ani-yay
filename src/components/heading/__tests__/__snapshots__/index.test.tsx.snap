// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render Heading component 1`] = `
<div>
  <h3
    class="sc-bczRLJ jYqVQl"
  >
    I am a test
  </h3>
</div>
`;
